bpf/alignchecker/bpf_alignchecker.c

```diff
@@ -43,6 +43,7 @@ int main(void)
 	DECLARE(struct, event_config, iter);
 	DECLARE(struct, tetragon_conf, iter);
 	DECLARE(struct, cgroup_tracking_value, iter);
-
-	return 0;
+	
+	// avoid unused variable warning/error
+	return iter * 0;
```

Comment:
Thanks for that fix! It seems fine. 

It seems that you should also run `make format`.

Reply:
Done `make format` and signed off.

(Had to `sudo dnf -y install clang-tools-extra` to bring in `clang-format` on RHEL8)